lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
```

Comment:
Note that the example in the comment above already includes the closable attributes, but doesn't quite match what is in the foundation 6 docs: http://foundation.zurb.com/sites/docs/close-button.html